server/service/src/programs/patient/patient_updated.rs

```diff
@@ -49,12 +48,6 @@ pub fn update_patient_row(
     let name_upsert = patient_to_name_row(store_id, update_timestamp, patient, existing_name)?;
     name_repo.upsert_one(&name_upsert)?;
 
-    let name_link_row = NameLinkRow {
-        id: name_upsert.id.clone(),
-        name_id: name_upsert.id.clone(),
-    };
-    NameLinkRowRepository::new(con).upsert_one(&name_link_row)?;
```

Comment:
I sneaked this in from:
https://github.com/msupply-foundation/open-msupply/commit/fea8cefaabd5b0d50d78b5bf01863d8520f56dc3#r136857975